app/assets/stylesheets/base/icons.scss

```diff
@@ -0,0 +1,13 @@
+.crayons-icon {
+  vertical-align: bottom;
+  -moz-transform: rotate(360deg);
+
+  // If we ever have an icon with its own colors that we don't want to overwrite,
+  // lets use modifier class `.crayons-icon__default`. It will prevent applying
+  // currentColor from parent element.
+  &:not(#{& + --default}) {
```

Comment:
Do we have any instances of this yet? If not, I wouldn't add this yet because [YAGNI](https://martinfowler.com/bliki/Yagni.html).

Reply:
We may need it depending on our approach with for example Emojis OR social network icons (like right now we have a Twitter icon on article page using it's own brand color). 

But it's a fair point. Twitter example is a proof we may need it actually. But I'm not against simplifying some icons for sake of getting rid of that line of code. We just have to pick right approach.